product_docs/docs/efm/4/upgrading.mdx

```diff
@@ -26,9 +26,6 @@ Failover Manager provides a utility to assist you when upgrading a cluster manag
 
 3.  Modify the `.properties` and `.nodes` files for Failover Manager 4.7, specifying any new preferences. Use your choice of editor to modify any additional properties in the properties file (located in the `/etc/edb/efm-4.7` directory) before starting the service for that node. For detailed information about property settings, see [The cluster properties file](04_configuring_efm/01_cluster_properties/#cluster_properties).
 
-!!! Note
-    `db.bin` is a required property. When modifying the properties file, ensure that the `db.bin` property specifies the location of the Postgres `bin` directory.
-
```

Comment:
From commit message, there are a lot of required properties. My guess is this was added when the db.bin property was first introduced, and so an upgrade required someone to manually change it. That hasn't been the case for a long time.